src/Simplify_Select.cpp

```diff
@@ -35,18 +35,7 @@ Expr Simplify::visit(const Select *op, ExprInfo *bounds) {
              rewrite(select(x, y, intrin(Call::likely, y)), false_value) ||
              rewrite(select(x, intrin(Call::likely_if_innermost, y), y), true_value) ||
              rewrite(select(x, y, intrin(Call::likely_if_innermost, y)), false_value) ||
-
-             // Select evaluates both sides, so if we have an
-             // unreachable expression on one side we can't use a
-             // signalling error. Call it UB and assume it can't
-             // happen. The tricky case to consider is:
-             // select(x > 0, a/x, select(x < 0, b/x, indeterminate()))
-             // If we use a signalling error and x > 0, then this will
-             // evaluate indeterminate(), because the top-level select
-             // evaluates both sides.
-
-             rewrite(select(x, y, IRMatcher::Indeterminate()), y) ||
-             rewrite(select(x, IRMatcher::Indeterminate(), y), y))) {
+             false)) {
```

Comment:
Is the `|| false` necessary?

Reply:
I do this a bunch in the simplifier. It makes it easier to insert, remove, or comment out rewrite rules without worrying about messing up the closing parens and open brace. It's like using a trailing comma at the end of an initializer list.